app/views/projects/samples/attachments/concatenations/create.turbo_stream.erb

```diff
@@ -1,21 +1,31 @@
-<%= turbo_stream.remove "new_attachment_modal" %>
-
 <%= turbo_stream.update(
   "sample_files_modal",
   partial: "modal",
   locals: {
-    open: false
+    open:
```

Comment:
This can probably just be simplified to `open: @samples.errors.any? ? true : false`

Reply:
a8c391e95c0036aa40376d35bd1eaf43d4acf8f9

Reply:
Actually can it not just be simplified to `open: @samples.errors.any?` that should return a boolean

Reply:
Actually thinking this through, can you try wrapping this update in a conditional statement that if there are no errors you do a `turbo_stream.delete` and just remove the modal from the page?  I haven't tried it, but think it would work :)

Reply:
I can use turbo_stream.remove, but then I can't get the modal to show again.